lib/tests/streamlit/runtime/scriptrunner/script_runner_test.py

```diff
@@ -65,6 +65,14 @@
 text_latin = "complete! ð\x9f\x91¨â\x80\x8dð\x9f\x8e¤"
 
 
+# Workaround for https://github.com/pytest-dev/pytest/issues/12263:
```

Comment:
It looks like the issue is fixed in Tornado 6.4.1: https://github.com/pytest-dev/pytest/issues/12263#issuecomment-2154160054
If we increased the min version, would we need the workaround?

Reply:
Yep, but increasing the version of Tornado to >= 6.4.1 might not be an option based on how we try to be compatible with a wide range of dependency versions. We could increase it only in our tests, but that means that we cannot correctly run our min-version tests (to test with min supported tornado version).

Reply:
Oh ok yeah the version is also pretty new from June this year. Could you then at least update the comment with a brief explanation of what impact the change has on the test run? Will the test still be executed?

Reply:
The workaround might be the best solution until we feel good about raising the tornado min version for Streamlit to `>=6.4.1` (maybe in a year).

Reply:
Added a comment